content/docs/languages/_index.md

```diff
@@ -5,12 +5,17 @@ weight: 1
 nav_children: section
 ---
 
-Each language / platform has at least **quick start** and **tutorial** pages.
+Each language / platform has links to the following pages and more:
+
+- Quick start
+- Tutorials
+- API reference
+
 Select a language to get started:
 
 - [Android Java](android)
 - [C#](csharp)
-- [C++](cpp)
+- [C++](cpp), and gRPC Core
```

Comment:
Adding this makes it a bit inconsistent with the rest.

Maybe `[C / C++](cpp)` instead?

Reply:
@ejona86 - WDYT?

Reply:
Or I could just remove "gRPC Core" text I added here.

Reply:
I think we can just drop (not add) the "gRPC Core" text. Almost all the documentation is C++, and we don't really advertise the C apis anywhere. They aren't so much for users as for other implementors.

Reply:
👍 Reverting that line to what we had before.